docs/source/methods/random_numbers.rst

```diff
@@ -62,11 +66,126 @@ Note that equation :eq:`lcg-skipahead` has the same general form as equation :eq
+------------------------
+
+As this might be difficult to imagine, let's add an example. 
+
+* Let's assume seed value :math:`\xi_{i} = 1`.
+* From point (1), :math:`\xi_{i+1} = 7806831264735756412` according to :eq:`pcg_lcg`, which 
+  is::
+    
+    0110 1100 0101 0111 0110 1111 1010 1100 0100 0011 1111 1101 0000 0000 0111 1100
+
+  in binary representation.
+* The random xorshift from point (2) is performed in 3 steps
+   * First, bit shift by 59 positions means we keep first 5 bits of :math:`\xi_{i+1}`, which is
+     ``0 1101`` or :math:`13`, when represented as integer.
+   * Second, adding 5, we have to shift :math:`\xi_{i+1}` by :math:`13+5=18` bits - or, equivalently, 
+     throw away the last 18 bits, which yields::
+
+        01 1011 0001 0101 1101 1011 1110 1011 0001 0000 1111 1111
+        
+     which is :math:`29780697878783` as integer. 
+   * Finally, perform `bitwise XOR`_ with shifted bits and original :math:`\xi_{i+1}`::
+        
+        0110 1100 0101 0111 0110 1111 1010 1100 0100 0011 1111 1101 0000 0000 0111 1100
+        0000 0000 0000 0000 0001 1011 0001 0101 1101 1011 1110 1011 0001 0000 1111 1111
+        -------------------------------------------------------------------------------
+        0110 1100 0101 0111 0111 0100 1011 1001 1001 1000 0001 0110 0001 0000 1000 0011 
+
+  The resulting number is :math:`7806836819539398787` as integer.
+* After multiplication from point (3) we get 
+  :math:`7806836819539398787 \cdot 6364136223846793005 = 13112265920887772427` 
+  which is::
+    
+    1011 0101 1111 1000 0010 0000 0110 0010 0001 1110 1111 1001 1101 0101 0000 1011
+    
+  as bits.
+* Another xorshift is done as follows
+    * Shifted right by 43 positions is means keep the first :math:`64-43=21` bits::
+        
+        1 0110 1011 1111 0000 0100
+
+      or :math:`1490692` as int.
+    * Finally, there is another XOR::
+
+        1011 0101 1111 1000 0010 0000 0110 0010 0001 1110 1111 1001 1101 0101 0000 1011
+        0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0001 0110 1011 1111 0000 0100
+        -------------------------------------------------------------------------------
+        1011 0101 1111 1000 0010 0000 0110 0010 0001 1110 1110 1111 0110 1010 0000 1111 
+
+      And the output value as integer is :math:`13112265920887089679`.
+* Point (5), convert the value to double from interval :math:`[0, 1)` as 
+  :math:`13112265920887089679\cdot 2^{-64} = 0.710817`, which is the output of the generator.
+
+
+
+
 .. only:: html
 
    .. rubric:: References
 
+.. [Brown] Forrest B. Brown and Yasunobu Nagaya, *The MCNP5 Random Number Generator*, LA-UR-02-3782, Los Alamos National Laboratory, available online at: https://laws.lanl.gov/vhosts/mcnp.lanl.gov/pdf_files/anl-rn-arb-stride.pdf
+
+.. [LEcuyer] Pierre L'Ecuyer, *Tables of linear congruential generators of different sizes and good lattice structure*, Math. Comp. 68 (1999), 249-260, available online at: https://doi.org/10.1090/S0025-5718-99-00996-5
```

Comment:
These two references were present only as links and link to Brown was broken. 
I added more detailed references, this time the link hopefully won't rot